c++/src/kj/compat/http.c++

```diff
@@ -2594,12 +2663,167 @@ private:
 
     static constexpr byte FIN_MASK = 0x80;
     static constexpr byte RSV_MASK = 0x70;
+    static constexpr byte RSV1_MASK = 0x40;
     static constexpr byte OPCODE_MASK = 0x0f;
 
     static constexpr byte USE_MASK_MASK = 0x80;
     static constexpr byte PAYLOAD_LEN_MASK = 0x7f;
   };
 
+  class Context {
```

Comment:
Went with `ZlibContext` since `Compression` has been thrown around a lot already: https://github.com/capnproto/capnproto/pull/1505/commits/408d58a67849795a076ef5653bb89d10e37f0822